main.go

```diff
@@ -80,7 +81,9 @@ type Options struct {
 	Status struct{} `cli:"status"`
 	Unseal struct{} `cli:"unseal"`
 	Seal   struct{} `cli:"seal"`
-	Env    struct{} `cli:"env"`
+	Env    struct {
+		Format string `cli:"--format"`
+	} `cli:"env"`
```

Comment:
Instead of a single flag with an enumerated value, can we just do `--bash` and `--fish` flags, and error if more than one is set?

Reply:
Sure, but since we explicitly don't want to set two formats, wouldn't going with this "format" option be the cleaner approach?

Reply:
I was thinking more from the operator / caller perspective.

```
safe env --format bash
```

seems weird and clunky to me, compared to

```
safe env --bash
```

GNU core utils is chock full of examples of flags that make no sense together, but are still defined as composable flags.

Reply:
Good point. Will change it either today or tomorrow :)